### DISCUSSION TITLE
Create works although NOT_NULL constraint

### DISCUSSION THREAD

**Discussion by johannespn** [prisma/prisma#10414] [Q&A] [answered]
Hey,

I created a database via prisma (v3.3.0) using this migration script:
```
-- CreateTable
CREATE TABLE "account" (
    "name" TEXT NOT NULL,
);
```

Unfortunately, creating an account entry with an empty string like this still works:
```
prisma.account.create({
      data: {
        name: ""
    },
})
```

Database structure is showing that `NULL` is allowed on other fields, but not on this one.

Can somebody help me on this?
Thanks

**Comment by johannespn** [comment, marked as answer]
My bad, "" is valid in sql context

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
